src/icons/IconIndex.ts

```diff
@@ -192,6 +192,7 @@ export { MonitorDuotoneIcon } from './MonitorDuotoneIcon';
 export { MonitorIcon } from './MonitorIcon';
 export { NightDuotoneIcon } from './NightDuotoneIcon';
 export { NightIcon } from './NightIcon';
+export { PaletteIcon } from './PaletteIcon';
```

Comment:
I think you need to add the duotone icon here too. 😁

Reply:
@ckgardner Oh man, I'm a mess. Done it.